### ISSUE TITLE
EkstraklasaTV

### ISSUE THREAD

**Issue by garbatyprostak** [ramicspa/kodi#641]
Witam. Mam problem z odtwarzaniem magazynow z PolsatSport (Polsat Cafe np) , nie uzupelniane sa tez  skroty z ost spotkan ekstaklasy i magazyny z C+ jak to mialo miejsce kiedys) .Da sie to jakos naprawic czy taki juz urok tej wtyczki -pytam bo jestem zupełnym laikiem. Pozdrawiam
Live działa super BTW

**Comment by rafciu27** [comment]
Działa Ci live na VPN jakimś czy proxy z wtyczki?

**Comment by garbatyprostak** [comment]
Ja w uk wiec dziala mi bez, ale z moim Vpn (nord) tez dziala

**Comment by mbebe** [comment]
W PL Live z zaznaczonym proxy we wtyczce nie dziala. Jest wybor rozdzielczosci i nie odtwarza.

**Comment by garbatyprostak** [comment]
Do tego doszło jeszcze pytanie o kod co chwile (klikniecie czegokolwiek w Menu i poza nim ) .
Ramic nie dało by sie cos z tym zrobic bo to jest troche frustrujące;)

**Comment by rafciu27** [comment]
Dokładnie, lekka przesada z tym kodem. Kiedyś raz wpisałeś i miałeś spokój aż do następnej zmiany kodu, teraz przy każdym wejściu w zakładke trzeba wpisać na nowo. Frustrujące.

**Comment by tomasiek** [comment]
hm.. dziwne. Tylko na tej wtyczce czy na innych też? (oczywiście od ramica)
Jakieś szczegóły?
Platforma? Wersja kodi?
Log (najlepiej świeżo po uruchomieniu kodi i wejściu w menu).
U mnie na 2 systemach działa bez problemowo. Kod tylko przy pierwszym uruchomieniu...

**Comment by garbatyprostak** [comment]
nie sprawdzalem innych , teraz nie mam dostepu , wtyczki Ramica mam na SPMC na boxie.

**Comment by tomasiek** [comment]
Wtyczki działają  - sprawdzane przez mnie. Podaj logi.

**Comment by garbatyprostak** [comment]
Nie pisałem ,że nie działaja tylko na wtyczce Ekstraklasa musze co chwile wbijć kod a nie tylko po wejsciu na wtyczke

**Comment by garbatyprostak** [comment]
Nie wiem czy dobrze to zrobilem ,jestem w tym wzgledzie kompletnym laikiem
http://paste.semperpax.com/p7acrj4sm#line-285

**Comment by tomasiek** [comment]
https://github.com/ramicspa/kodi/issues/646#issuecomment-367351029

**Comment by garbatyprostak** [comment]
Danke zaraz sprawdze , a co z Cefe Futbol ,ktorego nie moge otworzyc?

**Comment by tomasiek** [comment]
Obecnie na stronie https://github.com/ramicspa/kodi nie ma kodu. Nie jest on też wymagany przez wtyczki. Wystarczy zaktualizować repozytorium i po instalacji nie pyta o kod. Podejrzewam że to nie jedyne zmiany...
